packages/verification-vue-component/src/locales/de.json

```diff
@@ -1,78 +1,123 @@
             },
             "notFound": {
-                "status": "Dokument konnte nicht verifiziert werden",
-                "message": "Dieses Dokument konnten wir in unseren Blockchain-Einträgen nicht finden.<br />Achtung: vielleicht wurde das Dokument unabsichtlich verändert oder es liegt ein technisches Problem vor.<br />Am besten kontaktieren Sie den Aussteller direkt."
+                "status": "Für dieses Dokument konnten wir keinen Eintrag finden",
+                "details": "Dieses Dokument ist uns nicht bekannt. Möglicherweise wurde es unabsichtlich modifiziert. <br><br> Mögliche Fehlerquellen:<br><ul><li>Der Aussteller hat die falsche Version publiziert.</li><li>Der Halter des Dokuments hat Ihnen die falsche Version des Dokuments zugestellt.</li><li>Die Datei wurde versehentlich verändert:</li></ul>durch \"Drucken als PDF\"<br>durch das Abspeichern in einem PDF-Bearbeitungsprogramm, das den Schutz aufgehoben hat.<br>durch Ausdrucken und Scannen<br><br>Bei Fragen kontaktieren Sie bitte den auf dem Dokument vermerkten Aussteller oder wenden Sie sich an unseren Support."
             },
             "revokedUnverified": {
-                "status": "Dieses Dokument wurde widerrufen. Der Aussteller ist nicht verifiziert.",
-                "message": "Dieses Dokument wurde tatsächlich vom untengenannten Aussteller zertifiziert, ist aber inzwischen widerrufen worden.<br />Möglicherweise ist es durch eine neuere Version ersetzt worden. Für weitere Informationen wenden Sie sich bitte an den Aussteller.<br /><strong>Vorsicht:</strong> Dieser Aussteller wurde nicht durch das Certifaction-Team überprüft.<br />Es handelt sich lediglich um eine selbstdeklarierte Identität.<br />Seien Sie vorsichtig und wenden Sie sich im Zweifelsfall direkt an den Aussteller."
-            },
-            "revoked": {
                 "status": "Dieses Dokument wurde widerrufen.",
-                "message": "Dieses Dokument wurde tatsächlich vom untengenannten Aussteller zertifiziert, ist aber inzwischen widerrufen worden.<br />Möglicherweise ist es durch eine neuere Version ersetzt worden.<br />Für weitere Informationen wenden Sie sich bitte an den Aussteller."
+                "details": {
+                    "tamperProof": "Fälschungssicher zertifiziert",
+                    "unverifiedIssuer": "Unverifizierter Aussteller",
+                    "blockchain": {
+                        "secured": "Durch Blockchain gesichert",
+                        "registering": "Wird auf Blockchain revoziert<span class=\"detail-footnote\">Für dieses Dokument wird gerade ein Revozierungseintrag auf der Ethereum-Blockchain publiziert. Dadurch wird der Status unabhängig verifizierbar.</span>"
```

Comment:
There is no space between `revoziert` and `<span`, is that on purpose?

Reply:
.detail-footnote is displayed on a new line anyway so it doesn't make a difference